### PR TITLE
feat: update celery flower (PSRE-1267)

### PR DESCRIPTION
**Pull request by nadeemshahzad**
This PR upgrades our Flower Dockerfile to use focal instead of xenial.  It also upgrades `redis` from 3.2.0 to 4.1.1, `celery` from 4.4.7 to 5.2.3 and `flower` from 0.9.5 to 1.0.0.

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: docker/build/flower/Dockerfile [hunk 1]
```diff
@@ -22,4 +22,4 @@ USER nobody
 # Mount a config here if you want to enable OAuth etc
 ADD docker/build/flower/flowerconfig.py /flowerconfig.py
 
-ENTRYPOINT [ "flower" ]
+ENTRYPOINT [ "celery" ]
```
Comment (adzuci) [inline docker/build/flower/Dockerfile:25]: Why is the entrypoint now celery?

**Comment by adzuci** [comment]
Could you add a description to your PR mentioning the focal upgrade and that flower v1 is not currently available via pip directly so we are temporarily using an egg?

Could also be worth a comment in the dockerfile saying we should remove the GitHub link in the future if possible.

**Comment by nadeemshahzad** [comment]
thanks, @adzuci, flower version `1.0.0` is available via pip. I've updated the PR to install flower through pip.

File: docker/build/flower/Dockerfile (hunk 1 above)
Comment (nadeemshahzad) [reply docker/build/flower/Dockerfile:25]: In the new releases of flower, we need to invoke `flower` via `celery`. It is mentioned in the doc that this change is effective from version `1.0.1` (  https://flower.readthedocs.io/en/latest/install.html#usage ) but seems like change is also there for `1.0.0` as I am not able to use `flower` command to launch its service.

Comment (adzuci) [reply docker/build/flower/Dockerfile:25]: Ah, I see, thanks for the docs link.

**Review by adzuci — APPROVED** [review]
:shipit: If there is a good way to test locally, it might be good to add a comment to the PR to explain how to test these changes.

**Comment by nadeemshahzad** [comment]
We've README.txt to run and test this locally https://github.com/openedx/configuration/blob/master/docker/build/flower/README.txt

**Pull request merged**